src/api_routes.rs

```diff
@@ -170,6 +170,7 @@ pub fn config(cfg: &mut web::ServiceConfig, rate_limit: &RateLimit) {
           .route("/join", web::post().to(route_post::<UserJoin>))
           // Admin action. I don't like that it's in /user
           .route("/ban", web::post().to(route_post::<BanPerson>))
+          .route("/banned", web::get().to(route_get::<GetBannedPersons>))
```

Comment:
ban and banned should probably both go under `admin/`, but I don't want to make any breaking changes at the moment.

Reply:
Adding a new endpoint `/admin/banned` is not a breaking change, as long as existing endpoints dont change. Though removing `GetSiteResponse.banned`, it should at least return an empty list for now.

Reply:
We can just wait on this PR then, and hold off on merging it until we get a bunch of `v4` API stuff done.

Reply:
Well if youre fine to wait with releasing until 0.15, we can merge it into main and just not cherry-pick to the 0.14 branch.

Reply:
hrm... lets do another patch release to get a few fixes in, then we can merge this after.